src/main/java/seedu/staffsnap/logic/commands/EditCommand.java

```diff
@@ -109,7 +109,7 @@ private static Applicant createEditedApplicant(
         List<Interview> updatedInterviews = editApplicantDescriptor
                 .getInterviews().orElse(applicantToEdit.getInterviews());
 
-        return new Applicant(updatedName, updatedPhone, updatedEmail, updatedPosition, updatedInterviews);
+        return new Applicant(updatedName, updatedPhone, updatedEmail, updatedPosition, updatedInterviews, null);
```

Comment:
Will setting `applicant.status` to `null` every time we edit the Applicant cause the status value to be reset when an Edit command is executed?

Reply:
have replaced this with a proper integration, so now it will retain its status when edited if untouched